dev/com.ibm.ws.webcontainer.servlet.3.1_fat/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/CDIListenersTest.java

```diff
@@ -129,7 +122,7 @@ public String getListenersURL(String operationName, String comment) throws Unsup
 
         // @formatter:off
         return
-            LISTENERS_SERVLET_URL + "?" +
+            LISTENERS_SERVLET_URL_FRAGMENT + "?" +
```

Comment:
Same question as previously, this looks like a consistent change so just want to understand why the URL changed.

Reply:
Answered above.